XenAdmin/Dialogs/NetworkingPropertiesPage.cs

```diff
@@ -178,10 +178,9 @@ public void RefreshButtons()
                 var clusteringEnabled = network.Connection.Cache.Cluster_hosts.Any(cluster =>
                     cluster.host.opaque_ref == host.opaque_ref && cluster.enabled);

```

Comment:
This `host.opaqueref` here will also complain if the host has not resolved. I think it would be better to write `pif.host.opaqueref`.